01_install_requirements.sh

```diff
@@ -24,65 +35,133 @@ fi
 
 # Install required packages
 # python-{requests,setuptools} required for tripleo-repos install
+if [ "${RHEL8}" = "y" ] ; then
+    sudo dnf install -y \
+        python36 \
+        python3-requests \
+        python3-setuptools
+    sudo alternatives --set python /usr/bin/python3
+
+    # TODO(russellb) - Install an rpm for this once OSP for RHEL8 is out
+    pushd ~
+    if [ ! -d crudini ] ; then
+        git clone https://github.com/pixelb/crudini
+    fi
+    pushd crudini
+    git pull -r
+    sudo pip3 install .
+    popd ; popd
+else
+    sudo yum -y install \
+        crudini \
+        python-pip \
+        python-requests \
+        python-setuptools
+fi
 sudo yum -y install \
-  crudini \
   curl \
   dnsmasq \
   golang \
   NetworkManager \
   nmap \
   patch \
   psmisc \
-  python-pip \
-  python-requests \
-  python-setuptools \
   vim-enhanced \
   wget
 
-# We're reusing some tripleo pieces for this setup so clone them here
-cd
-if [ ! -d tripleo-repos ]; then
-  git clone https://git.openstack.org/openstack/tripleo-repos
-fi
-pushd tripleo-repos
-sudo python setup.py install
-popd
+if [ "${RHEL8}" = "y" ] ; then
+    sudo subscription-manager repos --enable=ansible-2-for-rhel-8-x86_64-rpms
```

Comment:
Can we make this optional or check if the node is subscribed to allow it working on non-subscribed nodes(assuming they're using manually set repos).

Reply:
++

Reply:
If someone wants to do that as a follow-up, that's fine.  My intention is to test exactly as a customer would, using an official release, and registered/subscribed.  It works fine with my employee account.